rerun_loader_tfrecord.py

```diff
@@ -2,14 +2,14 @@
 """Example of an executable data-loader plugin for the Rerun Viewer for tensorboard files."""
 from __future__ import annotations
 
-import numpy as np
-from tensorflow.python.summary.summary_iterator import summary_iterator
 import rerun as rr  # pip install rerun-sdk
 import argparse
 import os
 
 def log_tb_summary_file(filepath: str) -> None:
     """Log a tensorboard summary file to Rerun."""
+    import numpy as np
+    from tensorflow.python.summary.summary_iterator import summary_iterator
```

Comment:
```suggestion
    # We defer importing numpy and summary_iterator since these are very slow
    # and only needed in the case where we actually have a .tfrecord file
    import numpy as np
    from tensorflow.python.summary.summary_iterator import summary_iterator
```